pkg/runner/common.go

```diff
@@ -37,7 +37,7 @@ func nextDataNetwork(lenNetworks int) (*net.IPNet, string, error) {
 	return subnet, gw, err
 }
 
-func zipRunOutputs(ctx context.Context, basedir string, input *api.CollectionInput, ow *rpc.OutputWriter) error {
+func gzipRunOutputs(ctx context.Context, basedir string, input *api.CollectionInput, ow *rpc.OutputWriter) error {
```

Comment:
Mind updating the title PR and description so that we keep the paper trail?

Reply:
Done